hudi-utilities/src/main/java/org/apache/hudi/utilities/deltastreamer/HoodieDeltaStreamer.java

```diff
@@ -326,8 +326,8 @@ public static class Config implements Serializable {
         description = "the min sync interval of each sync in continuous mode")
     public Integer minSyncIntervalSeconds = 0;
 
-    @Parameter(names = {"--spark-master"}, description = "spark master to use.")
-    public String sparkMaster = "local[2]";
+    @Parameter(names = {"--spark-master"}, description = "spark master to use, if not defined inherits (e.g. not using spark-summit command).")
```

Comment:
nit: `spark-submit`

Also, it's not very clear from the message, what does it inherit from? Would be good to mention the default port and compression coded at least.